### ISSUE TITLE
New factors in which enemies an AI attacks

### ISSUE THREAD

**Issue by PitchNeeded** [Sh0wdown/UnofficialCrusaderPatch#235]
Could it be made that AI lords with positive fear factor are more likely to attack enemies who use negative fear factor first, and AI lords with negative fear factor are more likely to attack enemies who use positive fear factor first.

I think that would be a good idea as strategically 'good' AIs would probably want to get rid of the evil lords first of their enemies to stop the cruelty to the region's peasants. Also 'evil' AIs might want to get rid of 'good' AI enemies first to stop their unhappy peasants leaving and joining a happier, less stressful castle.

**Comment by Lolasik011** [comment]
controversial decision on my

**Comment by J0J0Jo** [comment]
I think it could be a nice addition as it would be optional to enable this.

But I would like to see some changes to the way the AI gathers its troops and some increases in ranged units first. If possible, it would be awesome if the AI would avoid enemy castles on their way to the target of their attack. Then your suggestion would work even better.

**Comment by PitchNeeded** [comment]
Yep it should be made that even on vanilla AI attack target settings, the AI will never try to go around enemy castles on their way to attack another.
